src/how-tos/deploy-operate.md

```diff
@@ -176,6 +176,14 @@ Errors and misconfigured parts of the `zilla.yaml` file are detected by Zilla an
 
 This feature is demonstrated in the above Helm install command. Running a `helm update ...` with changes to the `zilla.yaml`, k8s will update the config map, which writes the new content into the running pods. Zilla will detect those file changes and load the new config.
 
+Zilla can load the `zilla.yaml` config from a remote source using the `zilla.engine.config.url` [Zilla property](../reference/config/zilla-cli.md#p-property). The auto reconfigure feature will still work when pulling the config remotely.
+
+The Zilla start command looks like this:
+
+```bash:no-line-numbers
+zilla start -P zilla.engine.config.url=http://example.com/zilla.yaml
+```
+
```

Comment:
In general, we shouldn't be documenting the `zilla.*` properties, as those are internal.

The zilla command has a `-c` or `--config` option to specify where to get `zilla.yaml` including a url like `http://example.com/zilla.yaml`. Let's document that instead of `-P ...`.

Reply:
updated to recommend the -c flag and added the example to the ref section